utilities/plugins/openai_plugin.py

```
@@ -102,7 +102,7 @@ def streaming_request_http(
                     message = json.loads(data)
                     token = self._parse_streaming_response(endpoint, message)
                     tokens.append(token)
-        except (requests.exceptions.RequestException, json.JSONDecodeError):
+        except requests.exceptions.RequestException, json.JSONDecodeError:
             LOGGER.error("Streaming request error")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)